### PR TITLE
Slombard/cha 255 tweak add success modal

### PR DESCRIPTION
**Pull request by 552020**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
-REACT_APP_BACKEND_URL=https://daniilbot-k9qlu.ondigitalocean.app
\ No newline at end of file
+REACT_APP_BACKEND_URL=https://daniilbot-k9qlu.ondigitalocean.app
+VITE_BACKEND_URL=https://daniilbot-k9qlu.ondigitalocean.app
```
Comment (lmangall) [inline .env:2]: Why is there 2 and they are the same ?

**Review by lmangall — COMMENTED** [review]
These changes seem so complicated, well done

**Comment by lmangall** [comment]
Unfortunately there are conflicts

File: .env (hunk 1 above)
Comment (552020) [reply .env:2]: I don't know. Probably we can give read of the first. The first one was from the first code (and was not working), the second one should work and is an env variable from VITE.

**Comment by 552020** [comment]
@lmangall

File: .env (hunk 1 above)
Comment (lmangall) [reply .env:2]: Anyway we have to start with a .env...

**Review by lmangall — APPROVED** [review]
LooksComplicateToMe      😬

**Pull request merged**